### ISSUE TITLE
[Suggestion]Let the train stop output the number of charged and drained battery packs to the circuit network

### ISSUE THREAD

**Issue by UntouchedWagons** [dewiniaid/BatteriesNotIncluded#3]
This way you can configure the train to not leave till it has 5 charged battery packs or whatever.

**Comment by dewiniaid** [comment]
I do not believe this is currently possible as the mod is currently implemented.  Technically, the "batteries" are just fancy fuel with a special type (similar to how nuclear fuel cells work)

As a workaround, you can use the Inventory Sensor mod, which will read the contents of the container it's facing -- which I believe includes fuel for locomotives.
